presto-bytecode/src/main/java/com/facebook/presto/bytecode/MethodDefinition.java

```diff
@@ -211,6 +216,25 @@ public void visit(ClassVisitor visitor)
         visit(visitor, false);
     }
 
+    private int estimateCodeSize(BytecodeNode node)
```

Comment:
One thing I meant to ask about. This only counts the leaves as part of the estimated size (where `getChildNodes().isEmpty()` is `true`. This seems like it would undercount?

Do we also care about how constants like string constants take more space?

Reply:
Still wondering about this

Reply:
No it doesn't most others don't result in actual instructions. Anyway, super-hacky way to check so 128K. Also this limit is approximate so it should be fine.